patches/tModLoader/Terraria/ModLoader/Default/UnloadedPrefix.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using Terraria.Localization;
+
+namespace Terraria.ModLoader.Default;
+public sealed class UnloadedPrefix : ModPrefix
+{
+	public override LocalizedText DisplayName => Language.GetText("Mods.ModLoader.Unloaded");
+	public override IEnumerable<TooltipLine> GetTooltipLines(Item item)
+	{
+		UnloadedGlobalItem unloadedGlobalItem = item.GetGlobalItem<UnloadedGlobalItem>();
+		yield return new TooltipLine("UnloadedPrefix", $"{unloadedGlobalItem.modPrefixMod}/{unloadedGlobalItem.modPrefixName}") {
+			IsModifier = true,
+			OverrideColor = new(215, 123, 186)
+		};
```

Comment:
Please add screenshots of some unloaded prefixed item tooltips to the PR description.
Also include one of an unloaded item with an unloaded prefix

Reply:
At this time, if unloaded items retain prefixes they still do not display them, an image of a loaded item with an unloaded prefix has been attached.